doc/source/whatsnew/v0.23.2.txt

```diff
@@ -27,6 +27,8 @@ Performance Improvements
 - Improved performance of membership checks in :class:`CategoricalIndex`
   (i.e. ``x in ci``-style checks are much faster). :meth:`CategoricalIndex.contains`
   is likewise much faster (:issue:`21369`)
+- Improved performance of membership checks in :class:`Categorical`
```

Comment:
can you make this refernce this PR number (or just coming the what's new into 1 line either way)